postBuild

```diff
@@ -3,11 +3,7 @@
 # See https://mybinder.readthedocs.io/en/latest/config_files.html?#postbuild-run-code-after-installing-the-environment
 
 # Dependencies
-# - matplotlib: for MPL drawer
-# - pylatexenc: for MPL drawer
-# - pillow: for image comparison
-# - appmode: jupyter extension for executing the notebook
-pip install matplotlib pylatexenc pillow appmode
+pip install -r requirements-binder.txt
```

Comment:
why not just do `pip install -r requirements-dev.txt appmode`?

Reply:
because is long and mybinder is slow enough already .